### PR TITLE
Update Mason to check for cluster

### PR DESCRIPTION
**Pull request by sebastienvas**
Fixes https://github.com/istio/istio/issues/4860

### CODE REVIEW

File: boskos/cmd/mason/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,11 @@
-FROM alpine
+FROM google/cloud-sdk:alpine
 
-RUN apk update && apk upgrade && apk add ca-certificates && rm -rf /var/cache/apk/*
+RUN apk update \
+  && apk upgrade \
+  && apk add ca-certificates && rm -rf /var/cache/apk/* \
+  && gcloud components install kubectl -q --no-user-output-enabled
+
+RUN gcloud components install kubectl -q
```
Comment (chxchx) [inline boskos/cmd/mason/Dockerfile]: Is the repeatition intentional? The same command is included in the previous RUN clause

Comment (sebastienvas) [reply boskos/cmd/mason/Dockerfile]: you are right thanks!

**Comment by sebastienvas** [comment]
@chxchx  PTAL

**Pull request merged**